src/applications/common/schemaform/fields/ArrayField.jsx

```diff
@@ -17,6 +19,20 @@ import { errorSchemaIsValid } from '../validation';
 const Element = Scroll.Element;
 const scroller = Scroll.scroller;
 
+const isEmpty = (collection) => {
+  if (collection === undefined) {
+    return true;
+  }
+  if (typeof collection !== 'object') {
+    return false;
+  }
+  return _.every(isEmpty, Object.values(collection));
+};
+
+const notYetAnswered = (item, index, errorSchema) => {
+  return isEmpty(item) || !errorSchemaIsValid(errorSchema);
+};
```

Comment:
This helps to determine whether empty items with schemas that do not contain any required fields are displayed properly. This scenario is encountered if a user "adds another" saves the form, and returns later to continue filling the form out.
<img width="610" alt="screen shot 2018-05-30 at 5 56 57 pm" src="https://user-images.githubusercontent.com/16051172/40755597-db970902-6433-11e8-925e-fcefedfcfd52.png">

Reply:
Ok, that's fine for now. Can we make a ticket to go through the forms and update these cases to have at least one required field? I think it's a bad practice to have array fields where there are no required fields in the items. It encourages bad data being sent over. Once we do that, I think we could simplify how this works.

Reply:
Thanks @jbalboni, issue created https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10732.